src/pages/login/LoginPage.js

```diff
@@ -25,15 +25,14 @@ function LoginPage() {
         headers: {
           "Content-Type": "application/json",
         },
-        credentials: "include",  // added this line
+        credentials: "include", // added this line
         body: JSON.stringify(userData),
       });
 
       const data = await response.json();
-
       if (response.ok && !response.error) {
         handleLogin(data);
-        navigate("/");
+        navigate("/create-deck");
```

Comment:
I think we want to nav to homepage on login. I assume this was here for testing purposes though :) totally fine, just remember to remove/revert things like this before creating a PR as part of cleaning up.

Reply:
Absolutely just for testing, will remove it